tests/test_folium.py

```diff
@@ -197,7 +209,8 @@ def test_map_build(self):
 
         # Standard map.
         self.setup()
-        rendered = [line.strip() for line in self.m._parent.render().splitlines() if line.strip()]
+        rendered = [line.strip()
```

Comment:
Please avoid style changes in the same PR. It makes the reviewing process a little bit harder.
(Style changes are sometimes welcomed though, but preferably in a different PR.)

Reply:
I reverted them. My formatter accidentally modified it...